nextjs-logs-manager/app/components/client_navbar_buttons.tsx

```
@@ -0,0 +1,44 @@
+  const pathname = usePathname();
+  const router = useRouter();
+
+  if (pathname == "/log_loading") {
+    // here i might change router.push for router.back so if the page is bigger i can go back from other routes
+    return (
+      <Button color="gray" onClick={() => router.push("/")}>
+        <HiOutlineChevronLeft className="size-6" />
+      </Button>
+    );
+  } else {
+    <></>;
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.

(lint/complexity/noUselessFragments)